pymongoexplain/commands.py

```diff
@@ -24,28 +24,29 @@
 Document = Union[dict, SON]
 
 class BaseCommand():
-    def __init__(self, dictionary):
-        if type(self) == BaseCommand:
-            raise NotImplementedError
-        self.command_document = self.convert_to_camelcase(dictionary)
-
-    def convert_to_camelcase(self, d: dict) -> dict:
+    def __init__(self, command_name, collection):
+        self.command_document = {}
+        self.command_name = command_name
+        self.collection = collection
+
+    def convert_to_camelcase(self, d):
+        if not isinstance(d, dict):
+            return d
         ret = dict()
         for key in d.keys():
             if d[key] is None:
                 continue
             new_key = key
-            if "_" in key:
+            if "_" in key and key != "_id":
```

Comment:
Hmm, why do we need to exclude `_id`? What happens when a user attempts to explain something with underscores in field names? Like this:
```python
coll.update_one({'field_name': 1}, {'$set': {'other_name': 2}})
```

Reply:
I will change this so it only doesn't do capitalization if the first char is an underscore. I excluded "_id" because it was getting capitalized to "Id"

Reply:
I propose that we do not modify (or recurse into) the user's documents at all with this function. We only need to deal with top-level command argument field names right?

Reply:
In some cases, like with update or the delete commands, you have to append arguments into a sub dictionary, so in that case I think it makes sense.